python/README.md

```diff
@@ -3,20 +3,23 @@ MXNet Python Package
 MXNet is a deep learning framework designed for both *efficiency* and *flexibility*.
 It allows you to mix the flavours of deep learning programs together to maximize the efficiency and your productivity.
```

Comment:
nitpick: I think this MXNet intro seems out of place here. If required, we can do something like this:
"This directory and nested files contain [MXNet's](https://github.com/apache/incubator-mxnet) Python package and language binding."

Reply:
Agreed and done.